releasenotes/notes/introduce-cel-condition-in-container-filters-a2d0e49028367a08.yaml

```
@@ -0,0 +1,27 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    List new features here, or remove this section.
+
+    This commit introduces container_include_condition which uses CEL to better select which containers should be monitored.
+
+    More information about CEL:
+
+    https://github.com/google/cel-spec
+    https://github.com/google/cel-go
+
+    Now we can have the following condition expressions in the configuration:
+
+    name == 'eu_gcr' || name == 'private_jfrog'
+    name == 'secret-container-dd' && image == 'docker-dd-agent'
+    name.matches('secret-container-.*') && image == 'docker-dd-agent'
+
+    In addition it is compatible with the exclude_pause_container option.
```

```suggestion
    The `container_include_condition` option is compatible with the `exclude_pause_container` option.
```
Maybe say a couple words about what "is compatible" means? Do you mean the two options don't interfere with each other, or that the settings for one affect the settings of the other? or?